homeassistant/components/vacuum/services.yaml

```diff
@@ -0,0 +1,131 @@
+turn_on:
+  description: Starts a new cleaning task.
+
+  fields:
+    entity_id:
+      description: Name of the botvac entity.
+      example: 'vacuum.xiaomi_vacuum_cleaner'
+
+turn_off:
+  description: Stop the current cleaning task and return to home.
+
+  fields:
+    entity_id:
+      description: Name of the botvac entity.
+      example: 'vacuum.xiaomi_vacuum_cleaner'
+
+stop:
+  description: Stop the current cleaning task.
+
+  fields:
+    entity_id:
+      description: Name of the botvac entity.
+      example: 'vacuum.xiaomi_vacuum_cleaner'
+
+locate:
+  description: Locate the vacuum cleaner robot.
+
+  fields:
+    entity_id:
+      description: Name of the botvac entity.
+      example: 'vacuum.xiaomi_vacuum_cleaner'
+
+cleaning_play_pause:
+  description: Pause the cleaning task or replay it.
```

Comment:
It sounds better yes, although I suppose it also starts the cleaning, if it's not cleaning already?